rpc_jobs/rpc_openstack.yml

```diff
@@ -142,93 +142,6 @@
     jobs:
       - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
 
-- project:
-    name: "rpc-openstack-liberty-premerge"
```

Comment:
This is a PR test, not a PM test. Are you sure you want to remove PR tests too?

Reply:
I'm not sure on the outcome, but it was a deliberate decision. No one really should be committing things to Liberty anymore other than the ones we've been doing to get these old gates to pass.

The last official release was Nov 10, 2016.